data/meson.build

```diff
@@ -13,6 +13,15 @@ foreach i : icon_sizes
     )
 endforeach
 
+daemon_desktop_config = configuration_data()
+daemon_desktop_config.set('EXEC_PATH', libexecdir+'-daemon')
```

Comment:
Is this config line needed? It doesn't appear to me that it's used in the .desktop file

Reply:
Good catch! This reminded me to also remove the `libexecdir` variable from the root `meson.build` file 👍